test/integration/fixtures/blocks/core__media-text__deprecated-v2.html

```diff
@@ -0,0 +1,12 @@
+<!-- wp:media-text {"mediaType":"image", "isStackOnMobile": true} -->
+<div class="wp-block-media-text alignwide is-stacked-on-mobile">
```

Comment:
```suggestion
<!-- wp:media-text {"mediaType":"image"} -->
<div class="wp-block-media-text">
```

This fixture isn't covering the old v2 version. That version defaulted `isStackedOnMobile` to `false`. We should remove the explicit setting of the attribute in the block delimiter and the incorrect CSS class. 

With those removed, the block's markup will match the results of the v2 deprecation's save function using the old v2 attributes definition (`isStackedOnMobile: false`). After regenerating the fixtures, the final serialized HTML will match the current version i.e. `isStackedOnMobile` will be set to `false` explicitly within the block delimiter.